services/src/main/java/org/exoplatform/addons/gamification/rest/RealizationsRest.java

```diff
@@ -194,60 +194,35 @@ private String computeXLSX(List<GamificationActionsHistoryRestEntity> gamificati
         String eventKey = "exoplatform.gamification.gamificationinformation.rule.title.";
         String actionLabelKey = "exoplatform.gamification.gamificationinformation.rule.description.";
         String domainTitleKey = "exoplatform.gamification.gamificationinformation.domain.";
-        String actionId = "-";
         String actionLabel = "-";
-        String domainTitle = "-";
         String domainDescription = "-";
-        if (ga.getAction() != null) {
-          if (ga.getAction().getType() == TypeRule.AUTOMATIC) {
-            actionId = getI18NMessage(locale, eventKey + ga.getAction().getEvent().replace(" ", ""));
-            if (actionId == null) {
-              actionId = ga.getAction().getEvent();
-            }
-            actionId = escapeIllegalCharacterInMessage(actionId);
-          } else {
-            actionId = escapeIllegalCharacterInMessage(ga.getAction().getEvent());
-          }
-        }
         actionLabel = getI18NMessage(locale, actionLabelKey + ga.getActionLabel());
         if (actionLabel == null && ga.getAction() != null) {
           actionLabel = escapeIllegalCharacterInMessage(ga.getAction().getTitle());
         } else {
           actionLabel = escapeIllegalCharacterInMessage(actionLabel);
         }
         if (ga.getDomain() != null) {
-          domainTitle = getI18NMessage(locale, domainTitleKey + ga.getDomain().getTitle().replace(" ", ""));
           domainDescription = getI18NMessage(locale, domainTitleKey + ga.getDomain().getDescription().replace(" ", ""));
-          if (domainTitle == null) {
-            domainTitle = ga.getDomain().getTitle();
-          }
           if (domainDescription == null) {
             domainDescription = ga.getDomain().getDescription();
           }
         }
-        domainTitle = escapeIllegalCharacterInMessage(domainTitle);
         domainDescription = escapeIllegalCharacterInMessage(domainDescription);
         sbResult.append(ga.getCreatedDate());
         sbResult.append(DELIMITER);
         sbResult.append(ga.getCreator() != null ? ga.getCreator() : ga.getEarner());
```

Comment:
we need to get the earner here! 
ga.getEarner() != null ? ga.getEarner() : ga.getCreator() ;